dateparser/conf.py

```diff
@@ -57,18 +59,21 @@ def _updateall(self, iterable):
             setattr(self, key, value)
 
     def replace(self, mod_settings=None, **kwds):
```

Comment:
Why is `kwds` now a `dict` that contains a `settings` key?

Reply:
To pass other keyword arguments to Settings class which is required to get different registry_key for settings with different locale/language arguments.